checkov/openapi/checks/resource/generic/SecurityOperations.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Dict, Any
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.checks.enums import BlockType
+from checkov.openapi.checks.base_openapi_check import BaseOpenapiCheck
+
+
+class SecurityOperations(BaseOpenapiCheck):
+    def __init__(self) -> None:
+        id = "CKV_OPENAPI_5"
+        name = "Ensure that security operations is not empty."
+        categories = [CheckCategories.NETWORKING]
```

Comment:
please double check which category should be used for this checks, alternatives could be `CheckCategories.APPLICATION_SECURITY` or even create a separate one

Reply:
@schosterbarak what do you think?

Reply:
I think it's more API security